src/tests/ftest/util/dmg_utils_base.py

```diff
@@ -55,6 +55,8 @@ def __init__(self, path, yaml_cfg=None):
         self.debug = FormattedParameter("-d", False)
         self.json = FormattedParameter("-j", False)
 
+        self.result = None
+
```

Comment:
This is no longer required as it is defined in the inherited `YamlCommand`->`SubProcessCommand`->`CommandWithSubCommand` class: https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/command_utils.py#L414

Reply:
Done. Thanks for letting me know.